internal/server/deputies.go

```diff
@@ -146,7 +135,7 @@ func deputies(client DeputiesClient, tmpl Template) Handler {
 			DeputyList: deputyList,
 		}
 		vars.DeputyList.PaProTeamSelection = paProTeamSelection
-		vars.ECMs = vars.getECMs(app.Teams, app.SelectedTeam)
+		vars.ECMs = vars.getECMs(app.SelectedTeam)
```

Comment:
Wouldn't it be easier to just change `ECMs` to be an array of `Member` rather than `Assignee` and just do `app.SelectedTeam.Members` instead of a function that loops through them all and turns them into assignees?